### DISCUSSION TITLE
Watch files

### DISCUSSION THREAD

**Discussion by targetlucked69** [nuxt/nuxt#16178] [Questions] [answered]
Hello, `watch` property is no longer available in Nuxt stable... what is the equivalent of it now? If we want to watch certain file for changes... thanks

**Comment by TheAlexLichter** [comment, marked as answer]
@walmartwarlord What kind of files do you want to watch / what is *not* triggering HMR but should?

**Reply by targetlucked69** [reply]
nvm ty
